section-3/exercises/vsnitsar-section3.js

```diff
@@ -0,0 +1,182 @@
+/*
+Exercise 1:
+1. Create an empty array numbers, then add values 10, 20, 30 using index positions.
+2. Create an array with different data types: a string, a number, and a boolean. Print its length.
+3. Create an array animals with at least 3 elements. Print the second element.
+4. Change the last element of animals to "Lion" and print the updated array.
+*/
+
+console.log("Exercise 1.1.")
+const numbers = [];
+numbers[0] = 10;
+numbers[1] = 20;
+numbers[numbers.length] = 30;
+
+console.log("Exercise 1.2.")
+const mixedArray = ['sea', 33, true]
```

Comment:
Do not forger to add **;** . There are several places where you forget to add **;**